syntax = "proto3";

message LiveFeedSub  {
  repeated string livequote = 6;  // daftar simbol yang akan disubscribe
}

message SymbolSubscribe {
  string userId = 1;              // ID pengguna
  LiveFeedSub subs = 2;          	// daftar livequote
  string key = 3;                 // wskey
}
